src/main/java/io/aiven/kafka/connect/opensearch/RecordConverter.java

```diff
@@ -412,4 +428,76 @@ public String toString() {
             return name().toLowerCase(Locale.ROOT);
         }
     }
+
+    public enum DocumentIDStrategy {
```

Comment:
Could you make this enum top level please?  (since it belongs to configuration as well)